plugins/c9.vfs.standalone/standalone.js

```diff
@@ -142,7 +142,7 @@ function plugin(options, imports, register) {
         },
         previewHandler.getProxyUrl(function() {
             return {
-                url: "http://localhost:" + options.options.port + "/vfs"
+                url: "http://" + options.options.host + ":" + options.options.port + "/vfs"
```

Comment:
using host like this doesn't work when launching with `0.0.0.0`

Reply:
Found a fix for 0.0.0.0 issue, and this will be merged soon.
I have seen this issue before, but didn't find the root cause then.
Thanks for fixing it!